packages/sdk/src/lib.ts

```diff
@@ -92,26 +110,36 @@ export class PhotonClient {
       redirect_uri: redirectURI,
       cacheLocation: 'memory'
     });
-    this.audience = audience;
-    this.uri = uri;
-    this.clinicalUrl = getClinicalUrl(uri);
+    this.audience = env ? lambdasApiUrl[env] : audience;
+    this.uri = env ? `${lambdasApiUrl[env]}/graphql` : uri;
+    this.clinicalUrl = env ? clinicalAppUrl[env] : getClinicalUrl(uri);
```

Comment:
I think you want the opposite. Especially if `env` has a default value.

Reply:
makes sense, changing